docs/source/user/installing.rst

```diff
@@ -273,33 +273,37 @@ well but were not tested. The use of ``x`` in a version number indicates all
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.51.x | 2020-08-12   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.19     | 0.34.x                       | 10.x              | 2019.5 <= version < 2020.0  |
-|        |              |                           |                            |                              | (9.x for aarch64) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.50.x | 2020-06-10   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.19     | 0.33.x                       | 9.x               | 2019.5 <= version < 2020.0  |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.49.x | 2020-04-16   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.18     | 0.31.x <= version < 0.33.x   | 9.x               | 2019.5 <= version < 2020.0  |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.48.x | 2020-01-27   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.18     | 0.31.x                       | 8.x               | 2018.0.5 <= version < ?     |
-|        |              |                           |                            |                              | (7.x for ppc64le) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.47.x | 2020-01-02   | 3.5.x <= version < 3.9;   | 1.15 <= version < 1.18     | 0.30.x                       | 8.x               | 2018.0.5 <= version < ?     |
-|        |              | version == 2.7.x          |                            |                              | (7.x for ppc64le) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
++----------++--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| Numba     | Release date | Python                    | NumPy                      | llvmlite                     | LLVM              | TBB                         |
++===========+==============+===========================+============================+==============================+===================+=============================+
+| 0.57.x    | TBC          | 3.7.x <= version < 3.12   | 1.18 <= version < 1.24     | 0.40.x                       | 11.x              | 2021.x                      |
```

Comment:
min Python and NumPy may need an update here, depending on what we decided for a schedule.

Reply:
I will assume Pyhon at 3.8 and NumPy at 1.19 --happy to update if that changes.